Global/AmountInputField.js

```diff
@@ -1,29 +1,66 @@
+// React
 import React from "react";
 import { SafeAreaView, StyleSheet, TextInput } from "react-native";
 
-const AmountInputField = ({ text, onChangeText }) => {
+const AmountInputField = ({ onChangeCountryFlag, countryFlag, amount, onChangeAmount, ieAmount, currency, onChangeCurrency }) => {
   return (
-    <SafeAreaView>
+    <SafeAreaView style={styles.inputContainer}>
       <TextInput
-        style={styles.input}
-        onChangeText={onChangeText}
-        value={text}
+        style={styles.flag}
+        onChangeText={onChangeCountryFlag}
+        value={countryFlag}
+      />
+      <TextInput
+        style={styles.amount}
+        onChangeText={onChangeAmount}
+        value={amount}
+        placeholder={ieAmount}
+      />
+      <TextInput
```

Comment:
Why are we not using CurrencyInputField component?

Reply:
This is something that we will discuss when we have a meeting today.
I added comment on the issue card as well, I think we are breaking down components too small pieces, it is not very efficient for styling them, or working in structure as well. I will definitely explain them and we can talk about it in the meeting!

Reply:
Notes: we will add text input as a global component, and get rid of currency and flag